### PR TITLE
Update to include Jet Energy Fractions, softMuon ID

### PR DESCRIPTION
**Pull request by ikesisog-atCMS**
Added for the SOS analysis:  Jet Energy Fractions, softMuon ID

### CODE REVIEW

File: CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py [hunk 1]
```diff
@@ -230,6 +236,8 @@ def makeAllElectrons(self, event):
     def process(self, iEvent, event):
         self.readCollections( iEvent )
         self.counters.counter('events').inc('all events')
+        self.counters.counter('events').inc('vetoed events')
+        self.counters.counter('events').inc('accepted events')
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py]: Why do you need this extra counters that are all filled with the same count?

Comment (ikesisog-atCMS) [inline CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py]: Actually these lines remained after I was doing some debugging.
The lines containing "" and "" can be removed.

Should I do it and make a new pull request?

S.

Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py]: can you then remove those lines, commit and push again to the same branch, so the pull request is updated?
Thanks

Comment (ikesisog-atCMS) [inline CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py]: Done.

S.

On 12/01/2014 03:12 PM, Giovanni Petrucciani wrote:

> In CMGTools/TTHAnalysis/python/analyzers/ttHLepAnalyzerSusy.py:
> 
> > @@ -230,6 +236,8 @@ def makeAllElectrons(self, event):
> >      def process(self, iEvent, event):
> >          self.readCollections( iEvent )
> >          self.counters.counter('events').inc('all events')
> > -        self.counters.counter('events').inc('vetoed events')
> > -        self.counters.counter('events').inc('accepted events')
> 
> can you then remove those lines, commit and push again to the same branch, so the pull request is updated?
> Thanks
> 
> ---
> 
> Reply to this email directly or view it on GitHub https://github.com/CERN-PH-CMG/cmg-cmssw/pull/153/files#r21090146.

## 

Stilianos Kesisoglou, PhD
Research Associate
.......................................................................
CMS Experiment at CERN

CERN, CH-1211 Genève 23, Switzerland, Mailbox: E21610, Office: 40/4-A12

tel: (+41) (0) 22 76 71573 | fax: (+41) (0) 22 76 67847 | mobile: (+41) (0) 76 26 34328
email: stilianos dot kesisoglou at cern dot ch 
.......................................................................
Institute of Accelerating Systems and Applications (IASA)
National and Kapodistrian University of Athens (NKUA)
P.O. Box 17214 
GR-10024 Athens, GREECE

Bldg. IX / Office 129
tel:  (+30) 210 725 7533 | fax:  (+30) 210 727 6747 | mobile: (+30) 693 836 9970
email: kesisoglou at iasa dot gr
.......................................................................
Web Page: http://www.kesisoglou.eu/

Skype:        Full Name:  Stilianos Kesisoglou
              Skype Name: kesisoglou
              Country:    US

**Pull request merged**